kyuubi-server/src/main/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilder.scala

```diff
@@ -71,7 +70,7 @@ class FlinkProcessBuilder(
     }
 
     buffer += "-cp"
-    val classpathEntries = new util.LinkedHashSet[String]
+    val classpathEntries = new java.util.LinkedHashSet[String]
```

Comment:
The following error occurs during unit test, so the specified util package is displayed
![image](https://user-images.githubusercontent.com/35604105/200552168-61786147-6d6a-4cf1-ac1d-a3ab4fa04ede.png)